src/components/PostDetails/PostDetails.scss

```diff
@@ -7,6 +7,10 @@
     text-align: right;
   }
 
+  &__visiblelist {
+    display: none;
+  }
+
```

Comment:
Visible list should be a modifier of list

```scss
&__list {
  ...
  &--visible {
    ...
  }
}
```

Also, `visiblelist`'s behavior is inverted! it is hiding the element when set instead of making it visible; refactor to invisible attribute or invert logic